Makefile

```diff
@@ -1,31 +1,31 @@
 PG_CONFIG   ?= pg_config
 
 EXTENSION = tinyint
-EXTVERSION = 0.1.1
+EXTVERSION = 0.1.2
 EXTSQL = $(EXTENSION)--$(EXTVERSION).sql
+EXTSQLTMP = $(EXTENSION).sql
 
 MODULES = tinyint
 OBJS = tinyint.o
-DATA_built = tinyint.sql
 DATA = uninstall_tinyint.sql
 DOCS = doc/tinyint.md
 REGRESS = tinyint
 
 SQL_IN = tinyint.sql.in
-EXTRA_CLEAN += $(SQL_IN) $(EXTSQL)
+EXTRA_CLEAN += $(SQL_IN) $(EXTSQL) $(EXTSQLTMP)
 
-USE_EXTENSION = $(shell $(pg_config) --version | grep -qE " 8\.|9\.0" && echo no || echo yes)
+USE_EXTENSION = $(shell pg_config --version | grep -qE " 8\.|9\.0" && echo no || echo yes)
```

Comment:
This does a partial revert of the changes in c47c39e0af36ee85e14969b58700bc3d34f16978, making it harder to compile this module for a non-default postgresql version.
Such as compiling for pgsql 10/11/12 from pgdg repos when the distro only provides postgresql 9.6 (Such as is the case for EL/Centos 7).

Reply:
Sadly it did not work on Ubuntu and could not figure out in other way. I thing there need to be more if/else conditions.